### PR TITLE
Add PPP for KI

### PR DESCRIPTION
**Pull request by darkdestry-t**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -13,8 +13,7 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 <img src="images/darkdestry-t.png" width="200px">
 
-[[github](http://github.com/darkdestry-t)]
-<!-- [[portfolio](team/johndoe.md)] -->
+[[github](http://github.com/darkdestry-t)][[portfolio](team/darkdestry-t.md)]
```
Comment (tanboonji) [inline docs/AboutUs.md:16]: The link should be `team/darkdestry-t` without the `.md`

**Review by tanboonji — CHANGES_REQUESTED** [review]

**Review by yaowei-soc — APPROVED** [review]
LGTM

File: docs/AboutUs.md (hunk 1 above)
Comment (yaowei-soc) [reply docs/AboutUs.md]: It works even with the .md tho

Comment (darkdestry-t) [reply docs/AboutUs.md]: Fixed

**Review by tanboonji — APPROVED** [review]
LGTM

**Pull request merged**